### ISSUE TITLE
NVDA rarely fails to start at the Windows login screen

### ISSUE THREAD

**Issue by haruncetinkaya306** [nvaccess/nvda#17453]
### Steps to reproduce:
Press the power button on your computer to start Windows. Windows starts, but NVDA does not start, even though it should start at the Windows login screen.
### Actual behavior:

Windows starts, but NVDA does not start.
### Expected behavior:

It should start at the Windows login screen, as it does most of the time.
### NVDA logs, crash dumps, and other attachments:
[nvda.log](https://github.com/user-attachments/files/17958426/nvda.log)
[nvda-old.log](https://github.com/user-attachments/files/17958433/nvda-old.log)

#### NVDA installed/portable/running from source:
Installed.
#### NVDA version:
2024.4.1
#### Windows version:
Windows 11, 24 H2.
#### Name and version of other software used to reproduce the problem:
None.
#### Other information about your system:
None.
#### Does the problem persist after you restart your computer?
Yes.
#### Have you tried other versions of NVDA? If so, please report their behavior.
It happens with previous versions as well.
#### Does the problem persist if NVDA add-ons are disabled?
I don't use add-ons.
#### Does the problem persist after running the COM Registry Fix Tool from the tools menu of NVDA?
The problem persists.

**Comment by Adriani90** [comment]
Is the NVDA failing to start at the logon screen when you have to enter the password? Or is it failing to start after you loged in with you password?

Are you sure that NVDA failed to start? Or is just the speech synthesizer that doesn't initialize?
You can check whether NVDA commands work (e.g. nvda+n to open the NVDA menu).

Maybe related to #10954.

**Comment by haruncetinkaya306** [comment]
@Adriani90
The section where NVDA does not start is exactly where we type the password. I understood that NVDA did not open from this; There is no sound indicating that NVDA is opening. Even though I use NVDA with the screen curtain open, when I experience this problem, the screen curtain does not open either. From here I understood that NVDA did not open. Sometimes, interestingly, when I open Narrator, it also starts talking. But today, when I opened Narrator, it did not talk either.

**Comment by seanbudd** [comment]
Welcome @haruncetinkaya306 ,

We are unable to process this issue as it stands.

Please reproduce this issue and provide a debug [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences. We need logging at a higher level.

If you can consistently reproduce it, we have further debugging instructions we'd like to try.

Kind Regards,
NV Access Software Developers

**Comment by Qchristensen** [comment]
User advised that this happens irregularly, but when it happened recently, he noticed the following error dialog after he had logged into Windows: "nvda.exe - Application Error The instruction at 0x0000000077633396 referenced memory at 0x0000000000000C3C. The memory cannot be read"

**Comment by CyrilleB79** [comment]
I still encounter this from time to time but never with the mentioned memory error dialog. This may  be unrelated.

I seem to encounter this sometimes after having launched NVDA with an empty alternate config folder (`-c` option) during test sessions.

**Comment by haruncetinkaya306** [comment]
But this error message occurred when NVDA did not start at the Windows login screen. Either this error is related to that, or I experienced both errors.

**Comment by Adriani90** [comment]
It might be that NVDA starts talking before the sound card driver is initialized, so the sound card wakes up after NVDA's speech is already finished. If you say narator has the same issue sometimes, this might make sense.

**Comment by SaschaCowley** [comment]
@Adriani90, The author already indicated that NVDA commands do not work, and the screen curtain is not activated despite being enabled in their secure config, which very strongly indicates that NVDA is not starting at all.

**Comment by harun-342** [comment]
I have this issue intermittently. I can't log it because I don't know when it will happen. If anyone has a log of this issue, can you please share it? It's getting really annoying.

**Comment by harun-342** [comment]
I have managed to reproduce the problem by turning the computer off and on several times. When I experienced the problem, I got the following error message again: nvda.exe - Application Error The instruction at address 0x0000000076F62C34 referenced memory at address 0x000000000000005. The memory cannot be read.

Press OK to terminate the program
I am leaving the logs below where I recorded the error, that is all I could do about this error.

[nvda.log](https://github.com/user-attachments/files/21024539/nvda.log)

[nvda-old.log](https://github.com/user-attachments/files/21024542/nvda-old.log)

**Comment by harun-342** [comment]
@seanbudd
Hello, I opened this topic but I had to change my Git account. Do I need to open a new topic or will everything continue on this topic?

**Comment by harun-342** [comment]
> Hoş geldin[@haruncetinkaya306](https://github.com/haruncetinkaya306),
> 
> Bu konuyu şu haliyle işleme koyamıyoruz.
> 
> Lütfen bu sorunu yeniden oluşturun ve davranışın hata ayıklama [günlük dosyasını sağlayın. Günlük düzeyinizin genel tercihlerde hata ayıklamaya ayarlandığından emin olun. Daha yüksek bir düzeyde günlük kaydına ihtiyacımız var.](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps)
> 
> Bunu tutarlı bir şekilde yeniden üretebiliyorsanız, denemek istediğimiz daha fazla hata ayıklama talimatımız var.
> 
> Saygılarımla, NV Access Yazılım Geliştiricileri

I revisited the issue in #18417.

**Comment by harun-342** [comment]
> Welcome [@haruncetinkaya306](https://github.com/haruncetinkaya306) ,
> 
> We are unable to process this issue as it stands.
> 
> Please reproduce this issue and provide a debug [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences. We need logging at a higher level.
> 
> If you can consistently reproduce it, we have further debugging instructions we'd like to try.
> 
> Kind Regards, NV Access Software Developers

I revisited the issue in #18417.

**Comment by harun-342** [comment]
@Adriani90
@seanbudd
@Qchristensen
@CyrilleB79
@SaschaCowley
I've shared the logs for this issue here again. Are these logs what you wanted? Did they help diagnose the error? Thank you.

**Comment by seanbudd** [comment]
@harun-342 please be patient, responding to this issue is on our Todo list

**Comment by seanbudd** [comment]
This needs further technical investigation

**Comment by garo-pro** [comment]
I face the same issue, although only when I start my pc without fast start. NVDA doesn't seem to start and I have no speech, however, Narator does work and if I enable it I see no error message either. I think this might could be system (or BIOS?) dependent. On my old machine with an Intel i5 7600 it worked fine, however on my AMD Ryzen 7 9700X this issue persists.

**Comment by harun-342** [comment]
@garo-pro
I was experiencing this error on my Intel i3 10th generation system as well. This appears to be an NVDA issue.
